tutorials/setup.md

```diff
@@ -1,3 +1,25 @@
+![CRI-O logo](../logo/crio-logo.svg)
+- [Build and install CRI-O from source](#build-and-install-cri-o-from-source)
+  * [Runtime dependencies](#runtime-dependencies)
+  * [Build and Run Dependencies](#build-and-run-dependencies)
+    + [Fedora 31 RHEL 7 CentOS](#fedora-31-rhel-7-centos)
+    + [RHEL8](#rhel8)
```

Comment:
I know that this wasn't you, but can you change references to `RHEL8` to `RHEL 8` here and below?